dataeng/resources/setup-exporter-properties.sh

```diff
@@ -0,0 +1,35 @@
+# Create destination directory
+mkdir -p /var/lib/jenkins/tmp/analytics-exporter/course-data
+
+# Install requirements into this (exporter) virtual environment
+pushd analytics-exporter/
+pip install -r github_requirements.txt
+pip install mysql-connector-python -e .
+popd
+
+# Configuration paths in analytics-secure
+SECURE_ROOT=${WORKSPACE}/analytics-secure/analytics-exporter
+CONFIG_PATH=${SECURE_ROOT}/${CONFIG_FILENAME}
+GPG_KEYS_PATH=${WORKSPACE}/data-czar-keys
+
+# Save virtualenv location and configuration paths
+echo "
+EXPORTER_VENV=${VIRTUAL_ENV}
+CONFIG_PATH=${CONFIG_PATH}
+GPG_KEYS_PATH=${GPG_KEYS_PATH}
+DATE=$(date +%d ${DATE_MODIFIER})
+ORG_CONFIG_PATH=${WORKSPACE}/${ORG_CONFIG}
+" > exporter_vars
+
+env | sort
+
+# Export job configuration files 
+exporter-properties \
+    --output-bucket=${OUTPUT_BUCKET} \
+    --output-prefix=${OUTPUT_PREFIX} \
+    --orgs="${ORGS}" \
+    --include=platform_venv \
+    --include=exporter_vars \
+    ${CONFIG_PATH} \
+    ${WORKSPACE}/${ORG_CONFIG} \
+    organizations
```

Comment:
nit: It may be good to add a note here about "organizations".  It is the "Directory where to save Jenkins property files", and is used by the fileBuildParameterFactory in the master job to generate parameters for each of the worker jobs.  (Took me a while to figure that out....)

Reply:
Yeah I second this comment.  I had to follow the file path on the actual Jenkins server to figure this one out.  The local path is determined by the config file in the secure repo.  I mention this because I updated the path because it was hard coded to a temporal mount on the admin Jenkins box (/mnt/tmp/analytics-exporter/course-data/...).  I think we should reference where this guy is defined.